src/extension/ip_extraction.c

```diff
@@ -70,6 +70,7 @@ static zend_string *_try_extract(const zval *nonnull server,
 static bool _parse_x_forwarded_for(
     zend_string *nonnull value, ipaddr *nonnull out);
 static bool _parse_plain(zend_string *nonnull zvalue, ipaddr *nonnull out);
+static bool _parse_plain_raw(zend_string *nonnull zvalue, ipaddr *nonnull out);
```

Comment:
I think the name of this one should be `parse_plain` and the previous one should be `parse_plain_private` or something around those lines

Reply:
It doesn't just parse private IP addresses but rather all of them and it's the same name used in the tracer ip extraction module, so let's keep it consistent.